km/km_snapshot.c

```diff
@@ -397,13 +397,6 @@ static inline int km_ss_recover_vcpus(char* notebuf, size_t notesize)
 static inline int km_ss_recover_km_monitor(char* notebuf, size_t notesize)
 {
    km_nt_monitor_t* mon = (km_nt_monitor_t*)notebuf;
-   char* base = (char*)(mon + 1);
-   if (mon->label_length > 0) {
-      km_warnx("label: %s", base);
-   }
-   if (mon->description_length > 0) {
-      km_warnx("description: %s", base + mon->label_length);
-   }
```

Comment:
I assume this was removed because it is annoying.
Nothing is fixed here, right?

Reply:
These were forcing the `/tmp/km_log.1` with every snapshot run, destroying the open on demand logic you did a while back. And I see no value. Also Asked John, he confirmed it was debug cruft